examples/siro_sandbox/controllers/fetch_baselines_controller.py

```diff
@@ -365,20 +367,31 @@ def _set_height(self):
         # Update the base
         self.get_articulated_agent().sim_obj.transformation = target_trans
 
-    def _start_fetch(self, human_pos, obj_pos):
-        """Start to fetch the object condition"""
+    def _human_unblock_robot(self, human_pos, robot_pos, obj_pos):
+        """Check if the human blocks the robot when the robot is near the target"""
         return (
-            np.linalg.norm(self.rigid_obj_interest.linear_velocity)
-            < START_FETCH_OBJ_VEL_THRESHOLD
-            and float(np.linalg.norm(np.array((human_pos - obj_pos))[[0, 2]]))
+            float(np.linalg.norm(np.array((human_pos - obj_pos))[[0, 2]]))
             > START_FETCH_OBJ_DIS_THRESHOLD
+            or float(np.linalg.norm(np.array((human_pos - robot_pos))[[0, 2]]))
+            > START_FETCH_ROBOT_DIS_THRESHOLD
+        )
+
+    def _start_fetch(self, human_pos, robot_pos, obj_pos):
```

Comment:
Can you explain this more? It looks like a boolean function that is checking something, whereas "start_fetch" sounds like you would call it to start fetching. Can you name it "_is_ok_to_start_fetch" or similar?